Sources/ClientRuntime/Retries/DefaultRetryStrategy/DefaultRetryStrategy.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct DefaultRetryStrategy: RetryStrategy {
```

Comment:
This is the implementation of the standard architecture retry strategy.

Despite the name "default", this will be used unchanged by the AWS SDK.